ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -713,14 +725,25 @@ func (ipfs *Connector) PinLsCid(hash *cid.Cid) (api.IPFSPinStatus, error) {
 	return api.IPFSPinStatusFromString(pinObj.Type), nil
 }
 
-func doPost(client *http.Client, apiURL, path string) (*http.Response, error) {
+func (ipfs *Connector) doPost(client *http.Client, apiURL, path string) (*http.Response, error) {
```

Comment:
if do post is no longer used for anything, then just rename `doPostCtx` to `doPost`.

Reply:
✅